internal/tests/execute.go

```diff
@@ -0,0 +1,75 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2022 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tests
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flowkit"
+	"github.com/onflow/flow-cli/pkg/flowkit/services"
+)
+
+type flagsTests struct {
+	// Nothing for now
+}
+
+var testFlags = flagsTests{}
+
+var ExecuteCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "run <filename>",
+		Short:   "Run a test script",
```

Comment:
Maybe more like: "Run Cadence tests"? anyway we can say they are cadence tests not some other type